data/Symbeeosis/observations.ttl

```diff
@@ -0,0 +1,40 @@
+## Brix (refractive index)
+
+<data/cosmetics> a qb:DataSet ;
+    rdfs:label    "Measurement  for cosmetics pilot";
+    qb:structure  <DSD/cosmetics> ;
+    sosa:hasFeatureOfInterest <feature/VineLeaves> .
+
+<DSD/cosmetics> a qb:DataStructureDefinition ;
+    qb:component
+    [qb:attribute sosa:hasFeatureOfInterest  ; qb:componentAttachment qb:DataSet],
+    [qb:attribute sdmx-attribute:unitMeasure ; qb:componentAttachment qb:MeasureProperty],
+    [qb:attribute bdg:assayMethod ; qb:componentAttachment qb:MeasureProperty ],
```

Comment:
Make `assayMethod, comparisonBaseline` Optional: `componentRequired false`

Reply:
Added it, but default is false